relayer/utils.ts

```diff
@@ -10,9 +7,15 @@ import {
   parseVaa,
   SignedVaa,
 } from "@certusone/wormhole-sdk";
-import { ParsedVaaWithBytes } from "./application";
+import { bech32 } from "bech32";
+import { deriveWormholeEmitterKey } from "@certusone/wormhole-sdk/lib/cjs/solana/wormhole/index.js";
```

Comment:
Shouldn't we import from the ESM version here instead?

Reply:
I think this is a bad idea since it'll probably break commonJS users.

Reply:
yeah, this is a problem with the wormhole sdk, they need to use the `exports` property I showed you earlier. I am forced to import one and the safest is to import cjs.